cluster-microservice-provider/src/main/java/io/silverware/microservices/providers/cluster/ClusterMicroserviceProvider.java

```diff
@@ -103,25 +114,20 @@ public void initialize(final Context context) {
          log.error("Cluster microservice initialization failed.", e);
          throw new SilverWareClusteringException(INITIALIZATION_ERROR, e);
       }
-   }
 
-   @Override
-   public Context getContext() {
-      return context;
-   }
-
-   @Override
-   public void run() {
       try {
          while (!Thread.currentThread().isInterrupted()) {
+            Thread.sleep(2000);
 
          }
       } catch (final Exception e) {
-         log.error("Cluster microservice provider failed.", e);
+         log.error("Cluster microservice provider failed.");
```

Comment:
The only method that can throw an exception is `Thread.sleep` and it is thrown when a shutdown of the platform is performed. So it is expected behavior and I removed log message completely.